grakn-engine/src/main/java/ai/grakn/engine/GraknEngineServer.java

```diff
@@ -338,28 +338,20 @@ private static void handleInternalError(Exception exception, Response response){
         response.type(ContentType.APPLICATION_JSON.getMimeType());
     }
 
-    private Pool<Jedis> instantiateRedis(GraknEngineConfig prop, String redisUrl, int redisPort) {
-        JedisPoolConfig poolConfig = new JedisPoolConfig();
-        // TODO Make this configurable in property file
-        poolConfig.setMaxTotal(128);
-        Optional<String> sentinelMaster = prop.tryProperty(REDIS_SENTINEL_MASTER);
-        // If sentinel is configured use a sentinel pool
-        // TODO Sentinel not fully supported yet
-        return sentinelMaster
-                .<Pool<Jedis>>map(s -> new JedisSentinelPool(s, ImmutableSet.of(String.format("%s:%s", redisUrl, redisPort)), poolConfig))
-                .orElseGet(() -> new JedisPool(poolConfig, redisUrl, redisPort));
-    }
-
-    private RedissonLockProvider instantiateRedissonLockProvider(int redisPort, String redisUrl) {
-        LOG.info("Connecting redisCountStorage client to {}:{}", redisUrl, redisPort);
-        Config redissonConfig = new Config();
-        redissonConfig.useSingleServer()
-                .setAddress(String.format("%s:%d", redisUrl, redisPort))
-                .setConnectionPoolSize(5);
-        return new RedissonLockProvider(Redisson.create(redissonConfig));
+    private RedisWrapper instantiateRedis(GraknEngineConfig prop) {
+        List<String> redisUrl = GraknEngineConfig.parseCSValue(prop.tryProperty(REDIS_HOST).orElse("localhost:6379"));
+        List<String> sentinelUrl = GraknEngineConfig.parseCSValue(prop.tryProperty(REDIS_SENTINEL_HOST).orElse(""));
+        boolean useSentinel = !sentinelUrl.isEmpty();
+        Builder builder = RedisWrapper.builder()
+                .setUseSentinel(useSentinel)
+                .setURI((useSentinel ? sentinelUrl : redisUrl));
+        if (useSentinel) {
+            builder.setMasterName(prop.tryProperty(REDIS_SENTINEL_MASTER).orElse("graknmaster"));
+        }
+        return builder.build();
     }
 
-    private static void printStartMessage(String host, String port) {
+    private void logStartMessage(String host, String port) {
```

Comment:
Very minor thing, I don't mind keeping static. But I prefer avoiding static in general. It's also a conceptual choice here: GraknEngineServer.logStartMessage doesn't make much sense when you have more than one engine (e.g. in tests). We log the message for a certain engine not of the class engine.